cascade-cli/src/Cascade/CLI/Data/HttpPort.hs

```diff
@@ -0,0 +1,40 @@
+{-|
+Module      : Cascade.CLI.Data.HttpPort
+Description : !!! INSERT MODULE SHORT DESCRIPTION !!!
+Copyright   : (c) 2020-2021 Cascade
+License     : MPL 2.0
+Maintainer  : Mohammad Hasani <the-dr-lazy@pm.me> (the-dr-lazy.github.io)
+Stability   : Stable
+Portability : POSIX
+
+!!! INSERT MODULE LONG DESCRIPTION !!!
+-}
+
+module Cascade.CLI.Data.HttpPort (HttpPort, Error(..), Errors, un, mk, def) where
```

Comment:
Let's move model types to `Cascade.CLI.Data.Model.XXX`. Also, I would like to suggest `FreePort` to this datatype. It just makes sure a port is free or busy. It doesn't care which port we mean.